wild-swimming-app/src/components/add-comment-form/Addcommentform.tsx

```diff
@@ -0,0 +1,79 @@
+import { postComments } from "./utils/addcomments";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { useAuth0 } from "@auth0/auth0-react";
+
+
+
+const AddCommentForm = () => {
+  const { user, isAuthenticated} = useAuth0();
+  console.log(user);
+  const params = useParams();
+  const location_Id =params.id;
+
+const[userName, setUserName]= useState('water_baby')
+const[submitting, setSubmitting]= useState(false);
+const[bodyMessage, setBodyMessage]= useState("")
+const [err, setErr] = useState("");
+const[commentsList, setCommentsList]= useState("");
+
+const[message, setMessage]= useState("");
+
+
+const handleSubmit = (event:any)=>{
+  event.preventDefault();
+    const inputComment ={
+     name:userName,
+     body:bodyMessage,
+    };
+setCommentsList((currentComments)=>{
+  return ([inputComment, ...currentComments])})
+                    setMessage("We have updated your comment");
```

Comment:
could this be 'added'? as we won't be doing PATCH requests for MVP

Reply:
yes sure ill change it to added